test/tests.cpp

```diff
@@ -226,62 +226,4 @@ void Test_BeamSearch_Gpt_Cuda() {
   for (auto model_path : c_tiny_gpt2_model_paths)
     Test_BeamSearch_Gpt_Cuda(model_path.first, model_path.second);
 }
-
-void Test_Phi2_Cuda() {
```

Comment:
why do you remove this?

Reply:
It was disabled code that was out of date and never used. I was just cleaning it up when I saw it.